yal/utils.py

```diff
@@ -75,7 +75,7 @@ def normalise_phonenumber(phonenumber):
             assert phonenumbers.is_valid_number(pn)
             return phonenumbers.format_number(pn, phonenumbers.PhoneNumberFormat.E164)
     except (phonenumbers.phonenumberutil.NumberParseException, AssertionError):
-        raise ValueError("Invalid phone number")
+        raise ValueError("Invalid phone number {}".format(phonenumber))
```

Comment:
Can we use a f-string here?

Reply:
Yes, I've added user phone number for debugging here.